src/Cake.Common/Tools/NuGet/NuGetAliases.cs

```diff
@@ -1100,5 +1101,33 @@ public static void NuGetInit(this ICakeContext context, string source, string de
             var runner = new NuGetIniter(context.FileSystem, context.Environment, context.ProcessRunner, context.Tools, resolver);
             runner.Init(source, destination, settings);
         }
+
+        /// <summary>
+        /// List packages on available from source using specified settings
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="packageId">The package Id</param>
+        /// <param name="settings">The settings.</param>
+        /// <returns>List of packages with their version</returns>
+        /// <example>
+        /// <code>
+        /// NuGetList("Cake", new NuGetListSettings {
```

Comment:
Would be good if the example showed that there are items being returned.

```csharp
var result = NuGetList("Cake", new NuGetListSettings {
    AllVersions = true
});

Reply:
I've updated the example code with something more descriptive